rpi_sensor_log/monitor.py

```diff
@@ -0,0 +1,49 @@
+"""
+Module for monitoring through Raspberry Pi.
+
+- Measuring from sensors.
+- Logging to Google Sheets.
+"""
+
+import argparse
+
+from rpi_sensor_log.sensor.adafruit_dht22 import AdafruitDHT22
+from rpi_sensor_log.notify.google_sheets import GoogleSheetsNotifier
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Monitor through Raspberry Pi.')
+    parser.add_argument(
+        '--google-sheets-url',
+        type=str,
+        required=True,
+        help='Google Sheets URL.',
+    )
+    parser.add_argument(
+        '--google-sheets-worksheet-name',
+        type=str,
+        required=True,
+        help='Google Sheets worksheet name.',
+    )
+    sensor_parser = parser.add_subparsers(
+        dest='sensor_type',
+        required=True,
+        help='Type of measuring sensor.',
+    )
+    adafruit_dht22_parser = sensor_parser.add_parser(
+        'adafruit_dht22',
+        help='Adafruit DHT22 sensor.',
+    )
```

Comment:
Interesting 😮 then what would be the command to execute the program?

Reply:
One can execute this using the command below.

```bash
python -m rpi_sensor_log.monitor \
    --google-sheets-url "https://docs.google.com/spreadsheets/d/1hLmX0G3rl2z99dtoUQMSXb6Jfe6e2AK0l4dOjuxHUjg/edit?gid=0#gid=0" \
    --google-sheets-worksheet-name "Temperature & Humidity (1-1)" \
    adafruit_dht22 \
    --pin-number 17 \
```